mbof/models.py

```diff
@@ -21,7 +21,6 @@ def currentUserObject():
     return User.objects.get(loginName=os.getenv('REMOTE_USER'))
 
 
-@python_2_unicode_compatible
```

Comment:
If you remove `@python_2_unicode_compatible` for this method, don't you need to rename `__str__()` to `__unicode__()` and update its code?